build.gradle

```diff
@@ -86,14 +86,10 @@ tasks.dokkaHtmlMultiModule.configure {
     }
 }
 
-task installGitHooks() {
-    def command = Runtime.getRuntime().exec("git config core.hooksPath")
-    def currentGitHookDir = command.inputStream.text.trim()
-    def hooksPath = "hooks"
-    if (currentGitHookDir != hooksPath) {
-        Runtime.getRuntime().exec("git config --local core.hooksPath ${hooksPath}")
-        Runtime.getRuntime().exec("chmod -R +x ${hooksPath}")
-    }
+task installGitHooks(type: Copy) {
```

Comment:
the old version caused issues with the main app as it seems to have changed the hookspath globally, rather than just for backpack (despite using `--local`). copying the file seems to be the standard approach used for installing githooks on apps, so switching to that. 
while this doesn't "uninstall" the custom hookpath this should still work if a custom path is set as the directory doesn't change, but it reduces the risk of a custom hookspath being changed accidentally.